client/src/components/CategoryDropdown.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+
+const CategoryDropdown = ({categoryDropFade}) => {
+
+  return(
+    <div className="categoryDropdown"
+    onClick={() => categoryDropFade()}>
+        <div className="columnArrow"/>
+        <ul className="categoryDropMenu">
+          <li className="row">
+          <div className="CatSuggestions">Pick a CATegory</div>
+          <div className="topBorder"></div>
+            <div className="catRows">
+               Floof
```

Comment:
Long term, it is better to dynamically render the categories based. But this works for now.

Reply:
ok yeah. I'll get that going here.